_includes/integrations/react-quick-start.md

```diff
@@ -0,0 +1,139 @@
+## TinyMCE React integration quick start guide
+
+The [Official TinyMCE React component](https://github.com/tinymce/tinymce-react) integrates TinyMCE into React projects.
+This procedure creates a [basic React application](https://github.com/facebook/create-react-app) containing a TinyMCE editor based on our [Basic example]({{site.baseurl}}/docs/demo/basic-example/).
+
+For examples of the TinyMCE integration, visit [the tinymce-react storybook](https://tinymce.github.io/tinymce-react/).
+
+### Prerequisites
+
+This procedure requires:
+* [Node.js (and npm)](https://nodejs.org/).
+* Access to `tinymce.min.js` on either:
+
+    * [Tiny Cloud]({{site.baseurl}}/cloud-deployment-guide/editor-and-features/).
+    * TinyMCE Self-hosted. See [Advanced installation choices]({{site.baseurl}}/general-configuration-guide/advanced-install/) for details on self-hosting TinyMCE.
+
+### Procedure
+
+1. On a command line or command prompt, install the [Create React App](https://github.com/facebook/create-react-app) package.
+
+    ```
+    $ npm install -g create-react-app
+    ```
+2. Create a new React project named `tinymce-react-demo`.
+
+    ```
+    $ create-react-app tinymce-react-demo
+    ```
+3. Change into the newly created directory.
+
+    ```
+    $ cd tinymce-react-demo
+    ```
+4. Install the `tinymce-react` package and save it to your `package.json` with `--save`.
+
+    ```
+    $ npm install --save @tinymce/tinymce-react
+    ```
+5. Using a text editor, open `/path/to/tinymce-react-demo/src/App.js` and replace the contents with:
+
+    ```js
+    import React from 'react';
+    import { Editor } from '@tinymce/tinymce-react';
+
+    class App extends React.Component {
+      handleEditorChange = (e) => {
+        console.log('Content was updated:', e.target.getContent());
+      }
+
+      render() {
+        return (
+          <Editor
+            initialValue="<p>This is the initial content of the editor</p>"
+            init={% raw %}{{{% endraw %}
+              height: 500,
+              menubar: false,
+              plugins: [
+                'advlist autolink lists link image charmap print preview anchor',
+                'searchreplace visualblocks code fullscreen',
+                'insertdatetime media table paste code help wordcount'
+              ],
+              toolbar:
+                'undo redo | formatselect | bold italic backcolor | \
+                alignleft aligncenter alignright alignjustify | \
+                bullist numlist outdent indent | removeformat | help'
+            {% raw %}}}{% endraw %}
+            onChange={this.handleEditorChange}
+          />
+        );
+      }
+    }
+
+    export default App;
+    ```
+    This JavaScript file will create the class `App` containing a TinyMCE editor configured to replicate the example on the [Basic example page]({{site.baseurl}}/docs/demo/basic-example/).
```

Comment:
This will generate an invalid link as `{{site.baseurl}}` is the docs base url, so you shouldn't include `docs` in the path.